static/app/views/settings/organizationAuth/providerItem.tsx

```diff
@@ -103,12 +95,7 @@ function ProviderItem({provider, active, onConfigure}: Props) {
         children({...props, renderDisabled: renderDisabledLock as any})
       }
     >
-      {({
-        hasFeature,
-        features,
-        renderDisabled,
-        renderInstallButton,
-      }: FeatureRenderProps) => (
```

Comment:
we were re-declaring the child render function as `FeatureRenderProps` which doesn't make a lot of sense. These are already defined somewhere else.